xmlua/document.lua

```diff
@@ -36,7 +37,8 @@ local metatable = {}
 function metatable.__index(document, key)
   return methods[key] or
     Serializable[key] or
-    Searchable[key]
+    Searchable[key] or
+    C14n[key]
```

Comment:
We don't need to do this if this is only the Document method.
We can add c14n methods to here.

Reply:
Could you move `c14n.lua` content to `document.lua`?
Because `c14n.lua` content isn't shared with multiple classes. `c14n.lua` content is only used by `Document`.

Reply:
done.